src/index.js

```diff
@@ -2,18 +2,18 @@ import express from "express";
 import cors from "cors";
 import dotenv from "dotenv";
 import Models from "./server/models";
-
+import guestRoute from "./routes/guest-route";
+import errorHandler from "../src/middlewares/errorHandler";
 dotenv.config();
 
 const app = express();
-app.use(express.urlencoded({extended: true}));
-
+app.use(express.json());
 app.use(cors());
-
+app.use('/api/guest', guestRoute);
```

Comment:
Let me suggest a better format

app.use('/api/', router);

Let this reference your index file in your route folder.